src/app/utils.test.ts

```diff
@@ -344,3 +346,148 @@ describe("shortenWalletAddress", () => {
     expect(result).toBe(expectedShortened);
   });
 });
+
+describe("searchPairs", () => {
+  const pairsList: any = [
+    {
+      name: "DEXTR/XRD",
+      token1: {
+        name: "Dexter",
+        symbol: "DEXTR",
+      },
+      token2: {
+        name: "Radix",
+        symbol: "XRD",
+      },
+    },
+    {
+      name: "ADEX/XRD",
+      token1: {
+        name: "Adex",
+        symbol: "ADEX",
+      },
+      token2: {
+        name: "Radix",
+        symbol: "XRD",
+      },
+    },
+    {
+      name: "CUPPA/XRD",
+      token1: {
+        name: "Cuppa",
+        symbol: "CUPPA",
+      },
+      token2: {
+        name: "Radix",
+        symbol: "XRD",
+      },
+    },
+  ];
+
+  test("should find pairs by full name", () => {
+    const result = searchPairs("DEXTR/XRD", pairsList);
+    expect(result).toEqual([
+      {
+        name: "DEXTR/XRD",
+        token1: {
+          name: "Dexter",
+          symbol: "DEXTR",
+        },
+        token2: {
+          name: "Radix",
+          symbol: "XRD",
+        },
+      },
+    ]);
+  });
+
+  test("should find pairs by symbol", () => {
+    const result = searchPairs("DEXTR", pairsList);
+    expect(result).toEqual([
+      {
+        name: "DEXTR/XRD",
+        token1: {
+          name: "Dexter",
+          symbol: "DEXTR",
+        },
+        token2: {
+          name: "Radix",
+          symbol: "XRD",
+        },
+      },
+    ]);
+  });
+
+  test("should find pairs by symbol and full name with different case", () => {
+    const result = searchPairs("cupPa", pairsList);
+    expect(result).toEqual([
+      {
+        name: "CUPPA/XRD",
+        token1: {
+          name: "Cuppa",
+          symbol: "CUPPA",
+        },
+        token2: {
+          name: "Radix",
+          symbol: "XRD",
+        },
+      },
+    ]);
+  });
+
+  test("should find pairs with space as delimiter", () => {
+    const result = searchPairs("DEXTR XRD", pairsList);
+    expect(result).toEqual([
+      {
+        name: "DEXTR/XRD",
+        token1: {
+          name: "Dexter",
+          symbol: "DEXTR",
+        },
+        token2: {
+          name: "Radix",
+          symbol: "XRD",
+        },
+      },
+    ]);
+  });
+
+  test("should find pairs with swapped order of tokens", () => {
```

Comment:
Can you add another test here to test whether this also works for names? 

e.g. -> "Radix dexter" -> should find "DEXTR/XRD" pair.